src/pages/round/[...number].tsx

```diff
@@ -75,57 +64,80 @@ export default function Round() {
     }
   }, [display, roundList, number]);
 
-  const Url = "/api/v1/bribedata/" + requestedRound;
-
   console.log(roundList.latest);
   console.log(number[0], requestedRound);
-
-  const { data: bribedata, isLoading } = useQuery(["bribedata", Url], () =>
-    fetchRoundData(Url)
-  );
-
-  if (isLoading) {
-    return <Center>Loading ... </Center>;
-  }
+  console.log(bribeData?.header);
+  console.log(bribeData?.bribelist);
 
   return (
     <>
-      <Center>
-        <Box>
-          <pre>
-            Round {requestedRound} -- <Link href={Url}>{Url}</Link>
-          </pre>
-        </Box>
-      </Center>
-      <Center>
-        <Box>
-          <pre>{bribedata?.version}</pre>
-        </Box>
-      </Center>
+      <Box>
+        <Center>
+          <Text fontSize="4xl">{bribeData?.header.roundName}</Text>
+        </Center>
+        <Center>
+          <Text fontSize="1.4rem">
+            Vote Start: {bribeData?.header.voteStart} - Vote End: {bribeData?.header.voteEnd} -{" "}
+            {bribeData?.header.timeRemaining}
+          </Text>
+        </Center>
+        <Center>
+          <Text fontSize="1.4rem">
+            Votes Total: {bribeData?.header.totalVotes.toLocaleString("en-us")} - on incentivized
+            Pools: {bribeData?.header.bribedVotes.toLocaleString("en-us")} - Total Voter:{" "}
+            {bribeData?.header.totalVoter} - Bribed Voter: {bribeData?.header.bribedVoter}
+          </Text>
+        </Center>
+        <Center>
+          <Text fontSize="1.4rem">
+            Total Incentives: {bribeData?.header.totalBribes.toLocaleString("en-us")} - avg/1kfB:{" "}
+            {bribeData?.header.avgPer1000}{" "}
+          </Text>
+        </Center>
+      </Box>
+
       {display === "cards" ? (
         <Center>
           <Wrap justify="center" padding="23px" margin="23px" spacing="50px">
-            {bribedata?.bribedata?.map((bribe: any, i: number) => (
+            {bribeData?.bribelist?.map((bribe: any, i: number) => (
               <Box
```

Comment:
This is also fine for now, but seems like we could leverage the chakra <Card> for this box (and put all the styles in the theme as a variant)